main.py

```diff
@@ -611,8 +613,17 @@ def modBook():
 
 def editMember():
     def addMember():
+        membername = addMemberFNameE.get() + " " + addMemberLNameE.get()
+        mycursor = mycon.cursor()
+        mycursor.execute(f"UPDATE TABLE CUSTOMERS SET member = 'Y' WHERE Customer Name LIKE '{membername}'AND Contact Number LIKE '{addMemberContactE.get()}'" )
```

Comment:
UPDATE TABLE customers SET member = 'y' WHERE name LIKE '{membername}' AND contact LIKE '{addMemberContactE.get()}